tests/contrib/django_util/test_views.py

```diff
@@ -152,8 +152,8 @@ def setUp(self):
         self.user = User.objects.create_user(
             username='bill', email='bill@example.com', password='hunter2')
 
-    @mock.patch('oauth2client.contrib.django_util.views.pickle')
-    def test_callback_works(self, pickle):
+    @mock.patch('oauth2client.contrib.django_util.views.jsonpickle')
+    def test_callback_works(self, jsonpickle):
```

Comment:
1. Suggest rename as `jsonpickle_mock`
1. Can you verify the `jsonpickle` mock gets called or that `jsonpickle.encode` gets called, etc.? (I realize the `pickle` mock wasn't verified, but "you break it, you but it" has been @nathanielmanistaatgoogle rule.)